test/test_garcia.jl

```diff
@@ -28,9 +29,12 @@ Y = Dataset(x)
     max_1_idx = Peaks.maxima(N,min_dist)
     max_2_idx = Peaks.maxima(N2,min_dist)
 
-    @test max_1_idx == max_2_idx
+    @test 30 ≤ max_1_idx[1] ≤ 35
+    @test 30 ≤ max_2_idx[1] ≤ 35
+    @test 69 ≤ max_1_idx[2] ≤ 75
+    @test 69 ≤ max_2_idx[2] ≤ 75
 
-    # # plot N-Statistic for the Lorenz system as in Fig. 2(a) in [^Garcia2005b]
+    # # plot N-Statistic for the Lorenz system as in Fig. 2(a) in [^Garcia2005a]
```

Comment:
I wonder, should we add a comment here that our code fails to reproduce the paper? If someone runs this code and doesn't get the same figure they might be like "do I have an incorrect installation?"

Reply:
Yeah, I also thought about it. How did you proceed in those cases for all the stuff you could not reproduce? Or maybe better to leave a note in the docs?!

Reply:
I write it down in both comments in the tests as well as in the docstrings. :)

Reply:
For example see my comment at the end of this docstring: https://juliadynamics.github.io/SignalDecomposition.jl/dev/#SignalDecomposition.ManifoldProjection . I am very explicit about what is wrong with the paper.